partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnNot.kt

```diff
@@ -5,12 +5,16 @@ package org.partiql.spi.function.builtins
 
 import org.partiql.spi.function.Function
 import org.partiql.spi.function.Parameter
+import org.partiql.spi.function.utils.FunctionUtils
 import org.partiql.spi.value.Datum
 import org.partiql.types.PType
 
+/**
+ * This is the boolean NOT predicate. Its name is obfuscated via the use of [FunctionUtils.SYSTEM_PREFIX_INTERNAL].
+ */
 internal val Fn_NOT__BOOL__BOOL = object : Function {
 
-    private var name = "not"
+    private val name = FunctionUtils.SYSTEM_PREFIX_INTERNAL + "not"
```

Comment:
There are some functions (like here and `IS_MISSING`, `IS_NULL`, etc) that do not directly use the `FunctionUtils.hidden` method. Is that because they haven't been migrated to use the new interface (from https://github.com/partiql/partiql-lang-kotlin/issues/1646)?

Reply:
Not necessarily. They don't have a bunch of overloads, so they don't need to directly implement the new interface.